test/src/module/config/configTest.c

```diff
@@ -144,6 +141,30 @@ testRun(void)
         TEST_RESULT_VOID(cfgOptionSet(cfgOptConfig, cfgSourceParam, varNewStrZ("cfg")), "set option config");
         TEST_RESULT_BOOL(cfgOptionTest(cfgOptConfig), true, "option valid and value not null");
 
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("option groups");
+
+        TEST_RESULT_BOOL(cfgOptionGroupValid(cfgOptGrpPg), false, "pg option group is not valid");
+        TEST_RESULT_BOOL(cfgOptionGroupIndexTest(cfgOptGrpPg, 0), false, "pg option group index 0 is not set");
+        TEST_RESULT_UINT(cfgOptionGroupIndexTotal(cfgOptGrpPg), 0, "pg option group index total is 0");
+
+        TEST_RESULT_VOID(cfgOptionValidSet(cfgOptPgPath + 1, true), "set pg1-path valid");
+        TEST_RESULT_BOOL(cfgOptionGroupValid(cfgOptGrpPg), true, "pg option group is valid");
+        TEST_RESULT_BOOL(cfgOptionGroupIndexTest(cfgOptGrpPg, 0), true, "pg option group index 0 is set");
+        TEST_RESULT_BOOL(cfgOptionGroupIndexTest(cfgOptGrpPg, 1), false, "pg option group index 1 is not set");
+        TEST_RESULT_UINT(cfgOptionGroupIndexTotal(cfgOptGrpPg), 1, "pg option group index total is 1");
+
+        TEST_RESULT_VOID(cfgOptionValidSet(cfgOptPgPath + 7, true), "set pg7-path valid");
+        TEST_RESULT_VOID(cfgOptionSet(cfgOptPgPath + 7, cfgSourceParam, VARSTRDEF("/path")), "set pg7-path");
+        TEST_RESULT_BOOL(cfgOptionGroupIndexTest(cfgOptGrpPg, 1), false, "pg option group index 1 is not set");
+        TEST_RESULT_BOOL(cfgOptionGroupIndexTest(cfgOptGrpPg, 7), true, "pg option group index 7 is not set");
```

Comment:
Shouldn't the comment be `pg option group index 7 is set`

Reply:
Fixed.